src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -297,6 +339,43 @@ export class SkyAgGridService implements OnDestroy {
       suppressDragLeaveHidesColumns: true
     };
 
+    defaultSkyGridOptions.columnTypes[SkyCellType.CurrencyValidator] = {
+      ...defaultSkyGridOptions.columnTypes[SkyCellType.Validator],
+      ...defaultSkyGridOptions.columnTypes[SkyCellType.Currency],
+      cellClassRules: {
+        ...defaultSkyGridOptions.columnTypes[SkyCellType.Validator].cellClassRules,
+        ...defaultSkyGridOptions.columnTypes[SkyCellType.Currency].cellClassRules
+      },
+      cellRendererFramework: undefined,
+      cellRendererSelector: getValidatorCellRendererSelector('sky-ag-grid-cell-renderer-currency-validator'),
+      cellRendererParams: {
+        /*istanbul ignore next*/
+        validator: (value: any) => {
+          // TODO: This needs to be tightened up around which currencies to allow and more specific matching.
+          /*istanbul ignore next*/
+          return !!`${value || ''}`.match(/^[^0-9]*(\d+[,.]?)+\d*[^0-9]*$/);
```

Comment:
Can this be overwritten? It seems like it should be able to be overwritten if not. Some consumers might want a amount within a certain range, for example.

Reply:
Yes, the validator can be overridden.